### DISCUSSION TITLE
How can I get the font family from a font?

### DISCUSSION THREAD

**Discussion by isaacfink** [opentypejs/opentype.js#602] [Q&A]
I am trying to load fonts for use with sharp js, I need the font family in a way that it will be recognized by sharp (using fontconfig) I don't know enough about fonts but is there a font family? not specific to languages but something universal that I could use, and if there is how can I get it with opentype?

**Comment by forresto** [comment]
Meaning this?

```js
const family = openTypeFont.getEnglishName("fontFamily");
const variant = openTypeFont.getEnglishName("fontSubfamily");
```
